importer/NlGemNl.py

```diff
@@ -0,0 +1,157 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class NlGemNl(Monument):
+
+    def set_adm_location(self):
+        """
+        Set administrative location.
+
+        Use the Municipality ID to resolve to wd item.
+        """
+        municipal_match = [
+            x["item"] for x in self.data_files["municipalities"]
+            if x["value"] == self.gemcode]
+        if len(municipal_match) == 1:
+            self.add_statement("located_adm", municipal_match[0])
+        else:
+            self.add_to_report("gemcode", self.gemcode, "located_adm")
+
+    def set_monuments_all_id(self):
+        """Map which column in table to ID in monuments_all."""
+        self.monuments_all_id = "{}/{}".format(self.gemcode, self.objnr)
+
+    def set_architect(self):
+        """
+        Set architect.
+
+        Only if it's wikilinked. There's a lot of free text
+        ones which cannot be matched to anything as they don't
+        have nlwp articles, only have initials not full names etc.
+        """
+        if self.has_non_empty_attribute("architect"):
+            if utils.count_wikilinks(self.architect) > 0:
+                wikilinks = utils.get_wikilinks(self.architect)
+                for wl in wikilinks:
+                    arch_q = utils.q_from_wikipedia("nl", wl.title)
+                    if arch_q:
+                        self.add_statement("architect", arch_q)
+
+    def set_inception(self):
+        """
+        Set building date.
+
+        Only if it's a single, full year, i.e. no "ca." or ranges.
+        """
+        if (self.has_non_empty_attribute("bouwjaar") and
+           utils.legit_year(self.bouwjaar)):
+            self.add_statement(
+                "inception", {"time_value": {"year": self.bouwjaar}})
+
+    def set_address(self):
+        """
+        Set Located at street address.
+
+        Validation: must contain digit.
+        Many instances of addresses within {{sorteer}} template,
+        so we strip those first.
+        """
+        street_address = False
+        if self.has_non_empty_attribute("adres"):
+            if len(utils.wparser.parse(self.adres).filter_templates()) == 1:
```

Comment:
A bit complicated to follow what is happening.

Could L63-L71 be rebuilt as:
```python
address_try = self.adres
if len(utils.wparser.parse(self.adres).filter_templates()) == 1:
    address_try = utils.wparser.parse(self.adres).filter_templates()[0].params[0]
if utils.contains_digit(address_try):
    street_address = address_try
```
   
Might be easier to read if rebuilt as

Reply:
Done.